pandas/tests/indexes/test_common.py

```diff
@@ -395,3 +396,15 @@ def test_astype_preserves_name(self, index, dtype, copy):
             assert result.names == index.names
         else:
             assert result.name == index.name
+
+
+@pytest.mark.parametrize("exp_arr, sort", [([0, 1, 4, 3], False), ([0, 1, 3, 4], True)])
```

Comment:
can you use the index fixture

Reply:
Done. Using the `sort` fixture. There are only two in conftest, and only this one is useful.